tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -96,19 +96,6 @@ elif [ $BUILD_DEVICE = "gpu" ]; then
         $GET_DOCKER_IMAGE_CMD --repository "onnxruntime-$IMAGE" \
             --docker-build-args="--build-arg BASEIMAGE=nvcr.io/nvidia/cuda:11.1.1-cudnn8-devel-${BUILD_OS} --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} --build-arg INSTALL_DEPS_EXTRA_ARGS=\"${INSTALL_DEPS_EXTRA_ARGS}\" --build-arg USE_CONDA=${USE_CONDA} --network=host" \
             --dockerfile Dockerfile.ubuntu_gpu_training --context .
-elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
```

Comment:
i think we may not want to delete this.
we also want a path to validate nvidia's published ubuntu based tensorrt container images.

Reply:
Then should we have a daily pipeline for it?

Reply:
yes, it's a TODO

Reply:
Do you need both tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt and tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt7_1?

Reply:
Yesterday Alisson/Liqun/Natalie had a meeting with me. I was asked to help clean up unused pipelines and pipeline yaml files . That's why I'm doing this.